tests/test_pkce.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2016 Google Inc. All rights reserved.
```

Comment:
Oh, you were serious about that? Sorry. I misinterpreted the ":-P" emoticon as meaning this was a joke suggestion. It does look weird, but if you prefer it, I'll do it.